lib/mcp/tool.rb

```diff
@@ -156,8 +158,7 @@ def call(**args)
       def input_schema_to_json
         return nil unless @input_schema
 
-        compiler = SchemaCompiler.new
-        compiler.process(@input_schema)
+        @input_schema.json_schema
```

Comment:
I'd suggest to also delete the `$schema` from the resulting object. MCP spec doesn't specify the JSON schema version (which is kinda weird), but just to conform with their specification, I'd remove JSON schema version.

```suggestion
        @input_schema.json_schema.tap { _1.delete(:$schema) }
```

Reply:
But I'm happy to see this PR here 😁